nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -436,8 +436,14 @@ bool RegulatedPurePursuitController::isCollisionImminent(
 bool RegulatedPurePursuitController::inCollision(const double & x, const double & y)
 {
   unsigned int mx, my;
-  costmap_->worldToMap(x, y, mx, my);
 
+  auto convertWorldToMap = costmap_->worldToMap(x, y, mx, my);
+  if (!convertWorldToMap) {
```

Comment:
These can be simply replaced to `if (!costmap_->worldToMap(x, y, mx, my))` since you don't make use of `convertWorldToMap` anywhere else than that statement. Also below

Reply:
Ah yeah ! That is a nice way to do it !